server.go

```diff
@@ -76,6 +89,7 @@ func main() {
 	s.Router.HandleFunc("/InstaCountAll", controllers.InstaCountAllHandler)
 	s.Router.HandleFunc("/NotificationCount", controllers.NotificationCountHandler)
 	s.Router.HandleFunc("/Pie/{id}", controllers.GeneratePieHandler(riskServiceEndpoint))
+	s.Router.HandleFunc("/codelookup", controllers.CodeLookup)
```

Comment:
I suggest "/CodeLookup" instead of "/codelookup".  This is more consistent with all of the other URLs (since they all use Pascal-case).